website/src/components/navbar/navbar-client.tsx

```diff
@@ -104,27 +108,33 @@ export function NavbarClient(
 									<div key={index}>
 										{_.isEmpty(section.links) && section.href ? (
 											<Link href={section.href} key={index}>
-												<Button variant="ghost">
-													<Typography size="md" weight="medium">
+												<Button variant="ghost" className="py-6 text-gray-800 hover:text-black">
```

Comment:
We shouldn't use colors like `black`, `white`, `green`, etc., but only those defined here: https://github.com/socialincome-san/public/blob/main/ui/src/globals.css

Reply:
replaced the mentioned colours with the colours defined in the theme, but got rid of hover effects for text, as colors in theme not sufficient for such effect.